### PR TITLE
papaparse implementation - read v2

### PR DESCRIPTION
**Pull request by if0s**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,17 @@ outgoing message.
 *   `Skip rows` - if you know that the incoming CSV file has certain number of headers you can indicate to skip them. The supported values are `None`, `First row`, `First two`, `First three` and `First four`.
 *   `Data columns` - here the values will be added dynamically based on the values in the `CSV Header` field. Here each data column will be listed with the name, Data Type and the Format to enable further configuration.
 
+### Read CSV file from URL v2
```
Comment (jhorbulyk) [inline README.md:46]: I don't think it makes sense to have the read action also exist as a trigger. I would be in favor of removing it (and depricating the existing trigger.)

File: component.json [hunk 2]
```diff
@@ -59,6 +101,47 @@
...
       }
     },
+    "read_action_v2": {
+      "main": "./lib/triggers/read_v2.js",
+      "title": "Read CSV attachment v2",
+      "fields": {
+        "url": {
+          "label": "CSV URL",
+          "required": false,
+          "placeholder": "http://my-url.com/foo.csv",
```
Comment (jhorbulyk) [inline component.json:111]: Perhaps use a steward URL as the placeholder?

File: component.json [hunk 3]
```diff
@@ -59,6 +101,47 @@
...
+          "note": "We will fetch this URL and parse it as CSV file, leave the field blank if you expect CSV attachment from previous step",
```
Comment (jhorbulyk) [inline component.json:112]: I would skip this and make the URL required.

File: component.json [hunk 4]
```diff
@@ -59,6 +101,47 @@
         "out": {}
       }
     },
+    "read_action_v2": {
+      "main": "./lib/triggers/read_v2.js",
+      "title": "Read CSV attachment v2",
+      "fields": {
+        "url": {
```
Comment (jhorbulyk) [inline component.json:108]: Move to `metadata.in`

**Review by jhorbulyk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (if0s) [reply README.md]: Trigger removed, old read deprecated

File: component.json (hunk 2 above)
Comment (if0s) [reply component.json]: There is no placeholder after field moved to metadata.

File: component.json (hunk 3 above)
Comment (if0s) [reply component.json]: Done

File: component.json (hunk 4 above)
Comment (if0s) [reply component.json]: All fields, except "Emit Behavior" moved to metadata

**Review by jhorbulyk — APPROVED** [review]

**Comment by stas-fomenko** [comment]
@jhorbulyk should we add `"deprecated": true` to 'old' action and trigger or get rid of them?

**Comment by jhorbulyk** [comment]
@stas-fomenko Will we gain the ability to remove no longer used dev-dependencies by removing the old actions? If so, then I guess we can delete over simply deprecate. Otherwise, I would start with just deprecating.

**Pull request merged**